app/templates/application.hbs

```diff
@@ -128,4 +124,8 @@
     <a href='mailto:help@crates.io'>Send us an email</a>
     <span class="sep">|</span>
     {{#link-to 'policies'}}Policies{{/link-to}}
-</div>
+</footer>
+
+<a href='https://github.com/rust-lang/crates.io' class='fork-me'>
+  <img src='/assets/forkme.png'/ alt="Fork me on GitHub">
```

Comment:
Just one more thing, github hid my comment on this because I made it on a commit and not the overall diff-- could you move the slash that's now between the `src` and the `alt` attributes to the end of the `img` tag so that it closes the tag? Then this PR should be good to go!

Reply:
@carols10cents Done!